src/vix_trigger_fomo_calc.py

```diff
@@ -0,0 +1,239 @@
+
+# Select from to which date you want to see what you could have made/lost
+# NB! Can not be today's date, and it will require some processing power with dates
+# far away from each other
+start_date = '2021-01-28'
+end_date = '2021-02-11'
+# Do not change these two
+days_back = 15
+SYMBOL = 'VIX'
+# Choose the stock you want to compare the vix trigger with (ticker)
+stock_symbol = "TSLA"
+
+
+def isNewHigh(high, data):
+    """
+    Returns True if the 'high' is higher than any of the highs in the data
+    array passed in. Otherwise returns False
+    Parameters:
+    high (float): Today's highest price for the VIX index
+    data (list): All price data for the VIX index 15 days back
+    """
+    highs = data.get("High")
+    for i in highs:
+        try:
+            if (float(i)) >= float(high):
+                return False
+        except ValueError:
+            return False
+    return True
+
+
+def isNewLow(low, data):
+    """
+    Returns True if the 'low' is lower than any of the lows in the data
+    array passed in. Otherwise returns False
+    Parameters:
+    low (float): Today's lowest price for the VIX index
+    data (list): All price data for the VIX index 15 days back
+    """
+    lows = data.get("Low")
+    for i in lows:
+        try:
+            if float(i) <= float(low):
+                return False
+        except ValueError:
+            return False
+    return True
+
+
+def isCurrentHigherThanOpen(current, today_open):
+    """
+    Simple check to see if the current price is greater than the open price
+    Parameters:
+    current (float): The current price for the VIX index
+    open (float): Today's opening price
+    """
+    return float(current) > float(today_open)
+
+
+def isCurrentLowerThanOpen(current, today_open):
+    """
+    Simple check to see if the current price is lower than the open price
+    Parameters:
+    current (float): The current price for the VIX index
+    open (float): Today's opening price
+    """
+    return float(current) < float(today_open)
+
+
+def vix_trigger():
+    """
+    Checks to see if the data follows one of the two rules.
+    Returns 0 if it signals a sell
+    Returns 1 if it signals a buy
+    Returns 2 if it does not signal anything
+    """
+
+    # Initialize the buy & sell indicators
+    buy, sell = False, False
+
+    # If the current price is higher than the open and today is a new 15-day low, then this is a SELL indicator
+    if isCurrentHigherThanOpen(current, newest_open) & isNewLow(newest_low, data):
+        sell = True
+
+    # If the current price is lower than the open and today is a new 15-day high, then this is a BUY indicator
+    if isCurrentLowerThanOpen(current, newest_open) & isNewHigh(newest_high, data):
+        buy = True
+
+    # If one of the two indicators is True, print out the given text
+    if buy | sell:
+        if buy:
+            return 1
+        if sell:
+            return 0
+    else:
+        return 2
```

Comment:
Nice addition! just wondering, why the if else test? Seems like this would cause the code to be unnessecary slow. 
Also, what does it mean when you return 2? Hold? 
I think you can remove the outer if test, and instead write something like:
```python3
if buy:
  # Buy
   return 1
elif sell:
   # Sell
   return 0
else:
   # Hold
   return 2
```